### ISSUE TITLE
Segmentation fault on musl systems

### ISSUE THREAD

**Issue by i80and** [MayhemYDG/iltorb#40]
I'm deploying iltorb into a Docker container running Alpine Linux, but the node process segfaults as soon as I call into iltorb.

There are two issues preventing it from working:

* glibc and musl aren't ABI-compatible: iltorb should build from source, but uses an incompatible binary instead.
* The symbol tinkering in `src/_memcpy.h` creates a hard dependency on glibc. The quick-and-dirty solution is to add an extra definition guard:
  ```c
  #if defined(__linux__) && defined(__x86_64__) && defined(__GLIBC__)
    __asm__(".symver memcpy,memcpy@GLIBC_2.2.5");
  #endif
  ```

**Comment by oohnoitz** [comment]
Published v1.3.4 with the extra guard definition. Hopefully fixes the issue with distros that uses musl instead of glibc.

**Comment by i80and** [comment]
Confirmed that this works. Thanks!
